tests/e2e/e2e_exec_test.go

```diff
@@ -602,6 +603,28 @@ func (s *IntegrationTestSuite) registerICA(owner, connectionID string) {
 	s.T().Logf("%s reigstered an interchain account on chain %s from chain %s", owner, s.chainB.id, s.chainA.id)
 }
 
+func (s *IntegrationTestSuite) execQueryEvidence(c *chain, valIdx int, hash string) (res evidencetypes.Equivocation) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+	defer cancel()
+
+	s.T().Logf("querying evidence %s on chain %s", hash, c.id)
+
+	gaiaCommand := []string{
+		gaiadBinary,
+		"q",
+		"evidence",
+		hash,
+	}
+
+	s.executeGaiaTxCommand(ctx, c, gaiaCommand, valIdx, func(stdOut []byte, stdErr []byte) bool {
```

Comment:
Nit: we might want to rename or rework `executeGaiaTxCommand` for cli commands that don't require validation of the tx hash...`keys` and `query` subcommands would fall under this category.

Reply:
Yes, I agree! I also moved the other API queries to the `querie.go` file in this PR #1780. The evidence API seems not to be working well. I opened an issue in the cosmos-sdk repo (https://github.com/cosmos/cosmos-sdk/issues/13444). We can only query by CLI now, my idea was to try to put all CLI exec commands in the same file, but we can move to `e2e_query_test.go`. WDYT?

Reply:
yeah sounds great 👍